src/compounds/ItemPage/ItemPage.jsx

```diff
@@ -29,7 +33,9 @@ const ItemPage = ({ title, titleStyle, description, descriptionStyle, img }) =>
 ItemPage.propTypes = {
   description: PropTypes.string.isRequired,
   descriptionStyle: PropTypes.shape({}),
-  img: Image.defaultProps.isRequired,
+  id: PropTypes.number.isRequired,
+  img: PropTypes.shape({}),
```

Comment:
why change the `img` propType definition?

Reply:
it had an error in the console with how it previously was

Reply:
ah. weird. can you list the src and alt props inside the shape though?

Reply:
done!